### ISSUE TITLE
[EPIC] Bloop Next Steps

### ISSUE THREAD

**Issue by kpodsiad** [scalacenter/bloop#1680]
Based https://contributors.scala-lang.org/t/bloop-next-steps/5554. These changes should simplify working with Bloop and ease the maintenance burden on any current and future contributors.

TODO:
- [x] publish bloop launcher artifacts for Scala 2.13 - #1646
- [ ] bump used Scala and sbt versions to the newest ones #1670, #1679 (blocked on #1662)
- [ ] use upstream Zinc instead of forked one - #1662
- [ ] remove modules which are no longer needed - #1650
- [ ] rewrite generation of the test projects to not happen on sbt load
- [ ] remove custom shading plugin and use [sbt-shading](https://github.com/coursier/sbt-shading) if possible
- [ ] migrate from sbt-release-early to sbt-ci-release
- [ ] add Scala steward
- [ ] move integrations to separate repositories, where they can be released independently.
- [ ] create a community suite (most likely some modification of buildpress) that would test out some of the more popular repositories.
- [ ] make benchmarks easier to run and use.
- [ ] investigate tests and try to make them less flaky

**Comment by tgodzik** [comment]
Thanks for putting it into an actual issue! I should have done it previously after the post on contributors.

**Comment by tgodzik** [comment]
Most of the things are done. but the last two ones we don't plan on working on msot likely/
